packers/jws.go

```diff
@@ -306,6 +306,12 @@ func recoverEthereumAddress(token *jws.Message, vm verifiable.CommonVerification
 	signedData := base64TokenParts[0] + "." + base64TokenParts[1]
 	hash := sha256.Sum256([]byte(signedData))
 	sig := token.Signatures()[0].Signature()
+	if len(sig) == 64 && alg == "ES256K-R" {
+		sig = append(sig, []byte{1}...)
+	}
+	if len(sig) == 64 && alg == "ES256K" {
```

Comment:
Are you sure that we can recover public key from signature with ES256K alg?

Reply:
we can recover ETH address for ES256K/ES256K-R
unit tests added